regulations/templates/regulations/chrome.html

```diff
@@ -33,6 +33,7 @@
 {% include "regulations/main-header.html" %}
 
 <div class="sub-head {% block sub-head-class %}{% endblock %}">
+    {% block sub-head-drawer-icons %}
     <a href="#content-wrapper" class="hidden-text">Skip to main content</a> <!-- skip to content link for screen readers-->
```

Comment:
Quoting CM Lubinski (2016-06-14 15:09:34)
> I like the idea of putting the skip-nav in it's own block; for most use cases, it should hang around even when the drawer icons aren't present.

Fixed :)


--
Aaron D Borden
Human and Hacker